client/signup/config/flows-pure.js

```diff
@@ -282,7 +282,10 @@ export function generateFlows( { getSiteDestination = noop, getPostsDestination
 	if ( config.isEnabled( 'signup/import-landing-handler' ) ) {
 		flows[ 'from-site' ] = {
 			steps: [ 'import-from-url', 'user', 'domains' ],
-			destination: ( { siteSlug } ) => `/settings/import/${ siteSlug }`,
+			destination: ( { importUrl, importSiteDetails, siteSlug } ) =>
+				`/settings/import/${ siteSlug }` +
+				( importSiteDetails.engine === 'wix' ? '/wix' : '' ) +
```

Comment:
This isn't working after a rebase -- looks like the `is-site-importable` check needs to be reinstated.

See: #27382

Reply:
Hacked back in w/ e02929d